affordable-housing/PublishAffordableExcelCSV.csv

```diff
@@ -0,0 +1,231 @@
+Coming Soon,,,,,


```

Comment:
Was this copied from the open data portal?

http://data.sanjoseca.gov/datasets/165367/affordable-housing-data/

If so, could you delete it? Instead, provide instructions or a script for downloading it.